authentication/handler.go

```diff
@@ -151,23 +148,18 @@ func (m *Middleware) decorateValidRequest(ctx context.Context, r *http.Request,
 }
 
 func jwtErrorToProblem(err error) error {
-	var ve *jwt_go.ValidationError
-	if errors.As(err, &ve) {
-		switch {
-		case ve.Errors&jwt_go.ValidationErrorMalformed != 0:
-			return custom_problems.MalformedToken()
-		case ve.Errors&jwt_go.ValidationErrorUnverifiable != 0:
-			// JWT KeyFunc errors
-			return custom_problems.UnverifiableToken()
-		case ve.Errors&jwt_go.ValidationErrorSignatureInvalid != 0:
-			return custom_problems.UnverifiableToken()
-		case ve.Errors&jwt_go.ValidationErrorExpired != 0:
-			return custom_problems.ExpiredToken()
-		case ve.Errors&jwt_go.ValidationErrorNotValidYet != 0:
-			return custom_problems.NotYetValidToken()
-		}
-
-		return custom_problems.InvalidToken(ve.Error())
```

Comment:
Does the JWT library expose any new validation errors? Perhaps that was handled by this case in the old version.